packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -1,42 +1,35 @@
 @import '../../../../common/common';
 
-:global {
-  .ms-PickerPersona-container {
-    @include focus-border();
-
-    display: inline-flex;
-    align-items: center;
-    background: $ms-color-neutralLighter;
-    margin: 1px;
-    cursor: default;
-    user-select: none;
-    max-width: 100%;
-
+.pickerPersonaContainer {
```

Comment:
remove everying .pickerPersona or any other block level names. You only need "container", or if there are multiple containers, "personaContainer".  CSS modules means everything is scoped to this component already.